src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    private static final int COLOR_FIGURE = Color.values().length;
+
+    public Color getRandomColor() {
+        int index = new Random().nextInt(COLOR_FIGURE);
```

Comment:
move random to the class variable, like: 
![image](https://user-images.githubusercontent.com/12864128/222900645-09b06d21-410b-46c1-ba60-74ce527fb9f6.png)